tests/FsCheck.Test/project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "compilerName": "fsc",
+    "compile": {
+      "includeFiles": [
+        "Helpers.fs",
+        "Random.fs",
+        "Property.fs",
```

Comment:
> Tests inside tests\FsCheck.Test are run with dotnet test, but only partially because Unquote doesnt support .net core atm

i sent a pr to unquote for .net core and is in review https://github.com/SwensenSoftware/unquote/pull/126
so after that you just add the files back